examples/layers/raw/shader_warm_up.dart

```diff
@@ -33,3 +33,141 @@ Future<void> main() async {
   ui.window.onBeginFrame = beginFrame;
   ui.window.scheduleFrame();
 }
+
+/// Default way of warming up Skia shader compilations.
```

Comment:
Should this just be deleted?

Reply:
Sure. I'll just remove it.

Reply:
I'm going to just delete the whole example. Filed https://github.com/flutter/flutter/issues/87132 to track just removing this API entirely